rust-runtime/aws-smithy-types-convert/README.md

```diff
@@ -0,0 +1,17 @@
+# aws-smithy-types-convert
+
+This crate provides utilities for converting between the types defined
+in [aws-smithy-types](https://docs.rs/aws-smithy-types) and types commonly used in other libraries.
+
+## Crate Features
+
+By default, no features are enabled. Using the conversions requires enabling one or more features:
+
+```toml
+[dependencies]
+aws-smithy-types-convert = { version = "VERSION", features = ["convert-chrono"] }
+```
+
+Currently, the follow conversions are supported:
+* `convert-chrono`: Conversions between `DateTime` and [chrono](https://docs.rs/chrono/latest/chrono/).
+* `convert-time`: Conversions between `DateTime` and [time](https://docs.rs/time/latest/time/).
```

Comment:
Should we also indicate that conversions to/from std SystemTime are provided by aws-smithy-types?